docs/user/rstudio/ide/guide/ui/command-palette.qmd

```diff
@@ -35,9 +37,9 @@ First and foremost, the command palette serves as a way to search for and invoke
 
 To find a command, enter a word or short sequence of characters from the command. For example, to create a new script, start typing `new scr`.
 
-![](https://www.rstudio.com/blog/rstudio-v1-4-preview-command-palette/new-script.png)
+![](images/new-script.png){fig-alt="A screenshot of the Command Palette and search results for 'new scr'."}
 
-You can keep typing to filter the list, or press Up/Down to choose a command from the list and then Enter to execute the chosen command. Commands are displayed with their bound keyboard shortcuts, if any, so that you know how to invoke the command directly with the keyboard next time.
+Keep typing to filter the list, or press Up/Down to choose a command from the list and then Enter to execute the chosen command. Commands are displayed with their bound keyboard shortcuts, if any, so that you know how to invoke the command directly with the keyboard next time.
```

Comment:
```suggestion
Keep typing to filter the list, or press Up/Down to choose a command from the list and then press **Enter** to execute the chosen command. Commands are displayed with their bound keyboard shortcuts, if any, so that you know how to invoke the command directly with the keyboard next time.
```

Reply:
Changed